Formula/fdroidserver.rb

```diff
@@ -17,23 +17,34 @@ class Fdroidserver < Formula
   depends_on "freetype"
   depends_on "jpeg"
   depends_on "libtiff"
+  depends_on "openjpeg" # For Pillow.
```

Comment:
Oh, it works if `openjpeg` isn't installed. So I guess it's finding `openjpeg` in the environment and trying to use it. Ugh.